src/dataframe/io.jl

```diff
@@ -861,6 +862,16 @@ function readtable(io::IO,
     return df
 end
 
+# Import  "StringEncodings" if available and set up decoder
+ENCODING_PACKAGE_READY = false
+try Pkg.installed("StringEncodings")
```

Comment:
This isn't a good way to implement optional dependencies, since it causes issues with precompilation.